src/app/store/technical-records/effects/technical-record-service.effects.spec.ts

```diff
@@ -515,5 +515,81 @@ describe('TechnicalRecordServiceEffects', () => {
     });
   });
 
-  // TODO: Unit tests for putUpdate and postProvisional methods
+  describe('updateTechnicalRecord$', () => {
+    it('should return a technical record on successfull API call', () => {
```

Comment:
So sorry, but it is one of those...
```suggestion
    it('should return a technical record on successful API call', () => {
```